### PR TITLE
Improve grammar

### PR DESCRIPTION
**Pull request by asdacosta**
## Because

Imrove clarity

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/introduction/introduction_to_web_development.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
...
-The work is typically very project focused and involves collaborating with a team that helps to coordinate the client's needs into the end product. The client could be a tech company, an organization, or a government. The work could involve front-end, back-end, or full-stack web development.
+The work is typically project-focused and involves collaborating with a team to coordinate the client's needs into the end product. The client could be a tech company, an organization, or a government. The work could involve front-end, back-end, or full-stack web development.
 
 Web development could be a good profession for you if you like solving logical problems, building useful things, and experimenting with new technologies.
-Web developers are in high demand, generally have a good work/life balance, and command comfortable salaries. Google your specific location to get a better sense of your local web development job opportunities.
+Web developers are in high demand, generally have a good work-life balance, and command comfortable salaries. Google your specific location to get a better sense of your local web development job opportunities.
```
Comment (JoshDevHub) [inline foundations/introduction/introduction_to_web_development.md:9]: `work/life` balance also occurs on lines 33 and 35. If we're going to hyphenate this instance, the other instances should be highlighted as well.

File: foundations/introduction/introduction_to_web_development.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ It acknowledges that you need a wide variety of skill sets and languages to reac
 
 The Odin Project is constantly evolving because of people like you who get further along in the curriculum and pay it forward by incrementally improving our content over time.
 
-Once you start to feel comfortable with the tools, start contributing to open source projects, like [contributing to The Odin Project](https://www.theodinproject.com/contributing)! The more you contribute, the more you will learn about what you can do, and the closer you will get to being hireable. These projects will also look great on your resume!
+Once you start to feel comfortable with the tools, start contributing to open-source projects, like [contributing to The Odin Project](https://www.theodinproject.com/contributing)! The more you contribute, the more you will learn about what you can do, and the closer you will get to being hireable. These projects will also look great on your resume!
```
Comment (JoshDevHub) [inline foundations/introduction/introduction_to_web_development.md:68]:
```suggestion
Once you start to feel comfortable with the tools, start contributing to open source projects, like [contributing to The Odin Project](https://www.theodinproject.com/contributing)! The more you contribute, the more you will learn about what you can do, and the closer you will get to being hireable. These projects will also look great on your resume!
```
I'm going to ask that this is reverted. `open source` occurs over 20 times throughout the curriculum, and not a single one of them is hyphenated. It'd be strange to introduce hyphenating it here.

It's also kind of up to preference whether you hyphenate it or not, and I wouldn't say the hyphenated version is strictly "correct". Some style guides (like [Google's](https://opensource.google/documentation/reference/comms/style-guide#quick_reference)) avoid hyphenating it, and if it's okay for an org that big, it's definitely okay for us.

**Review by JoshDevHub — CHANGES_REQUESTED** [review]
Thank you for contributing. Just have a couple small comments on this and we should be good to merge.

File: foundations/introduction/introduction_to_web_development.md (hunk 2 above)
Comment (asdacosta) [reply foundations/introduction/introduction_to_web_development.md]: > I'm going to ask that this is reverted. `open source` occurs over 20 times throughout the curriculum, and not a single one of them is hyphenated. It'd be strange to introduce hyphenating it here.
> 
> It's also kind of up to preference whether you hyphenate it or not, and I wouldn't say the hyphenated version is strictly "correct". Some style guides (like [Google's](https://opensource.google/documentation/reference/comms/style-guide#quick_reference)) avoid hyphenating it, and if it's okay for an org that big, it's definitely okay for us.

Ok @JoshDevHub

**Review by JoshDevHub — APPROVED** [review]
Thank you for contributing :rocket:

**Pull request merged**